packages/synapse-react-client/src/components/CannedRejectionDialog/CannedRejectionDialog.test.tsx

```diff
@@ -1,56 +1,46 @@
-import React from 'react'
 import { render, screen, waitFor, within } from '@testing-library/react'
-import { createWrapper } from '../../testutils/TestingLibraryUtils'
-import RejectDataAccessRequestModal, {
-  RejectDataAccessRequestModalProps,
-} from './RejectDataAccessRequestModal'
-import { rest, server } from '../../mocks/msw/server'
-import mockRejectionReasonsTableQueryResultBundle from '../../mocks/query/mockRejectionReasonsTableQueryResultBundle'
 import userEvent from '@testing-library/user-event'
-import { mockSubmittedSubmission } from '../../mocks/dataaccess/MockSubmission'
-import {
-  BackendDestinationEnum,
-  getEndpoint,
-} from '../../utils/functions/getEndpoint'
-import { DATA_ACCESS_SUBMISSION_BY_ID } from '../../utils/APIConstants'
-import { SubmissionState } from '@sage-bionetworks/synapse-types'
 import failOnConsoleError from 'jest-fail-on-console'
+import React from 'react'
 import { registerTableQueryResult } from '../../mocks/msw/handlers/tableQueryService'
+import { server } from '../../mocks/msw/server'
+import mockRejectionReasonsTableQueryResultBundle from '../../mocks/query/mockRejectionReasonsTableQueryResultBundle'
+import { createWrapper } from '../../testutils/TestingLibraryUtils'
+import {
+  CannedRejectionDialog,
+  CannedRejectionDialogProps,
+} from './CannedRejectionDialog'
+
+const MESSAGE_PREFIX = 'Thank you for submitting your request.\n'
+const MESSAGE_APPEND =
+  '\nIf you have questions, do not respond to this email address. Instead, reply to:\nact@sagebionetworks.org'
 
-const props: RejectDataAccessRequestModalProps = {
-  submissionId: mockSubmittedSubmission.id,
+const props: CannedRejectionDialogProps = {
   tableId: 'syn50683097',
   open: true,
+  error: null,
+  defaultMessagePrefix: MESSAGE_PREFIX,
+  defaultMessageAppend: MESSAGE_APPEND,
+  onConfirm: jest.fn(),
   onClose: jest.fn(),
 }
 
-const onServerReceivedUpdate = jest.fn()
-
-function renderComponent() {
-  return render(<RejectDataAccessRequestModal {...props} />, {
+function renderComponent(
+  propOverrides: Partial<CannedRejectionDialogProps> = {},
+) {
+  return render(<CannedRejectionDialog {...props} {...propOverrides} />, {
     wrapper: createWrapper(),
   })
 }
 
-describe('RejectDataAccessRequestModal', () => {
+describe('CannedRejectionDialog', () => {
```

Comment:
Logic to render the dialog + form was moved to `CannedRejectionDialog` component, so move the bulk of the test logic to a new test